fuse/dl/models/backbones/backbone_vit.py

```diff
@@ -17,13 +17,22 @@ class ProjectPatchesTokenizer(nn.Module):
     batch_size, num_tokens, token_dim
     """
 
-    def __init__(self, *, image_shape: Sequence[int], patch_shape: Sequence[int], channels: int, token_dim: int):
+    def __init__(
+        self,
+        *,
+        image_shape: Sequence[int],
+        patch_shape: Sequence[int],
+        channels: int,
+        token_dim: int,
+        strict: bool = True,
```

Comment:
Can you please specify in a comment what strict and what happens if it's False and what happens if the Image dimensions must be divisible by the patch size